### ISSUE TITLE
about performance in the continuous environment

### ISSUE THREAD

**Issue by SherrySwift** [alexfrom0815/Online-3D-BPP-PCT#17]
Firstly,  thank you so much for sharing the code. This is really excellent work! 
I train a model using the EMS scheme in a continuous environment, and then test the model on the 'setting13_continuous.pt' dataset that you provide. But I struggle to achieve the same performance as that in paper. In the paper, PCT & EMS in setting 3 achieves 66.6% utilization, but I can only achieve 62.6%.

Here is my training command:
python main.py --continuous --setting 3 --internal-node-holder 80 --leaf-node-holder 50 --sample-from-distribution --sample-left-bound 0.1 --sample-right-bound 0.5

The training goes on 20K iterations. The hyperparameters in tools.py remain exactly the same as default. (and I find that set 'shuffle' to False during training seems to get better performance)
Could you please give me some advice on improving the result? Thank you so much!

**Comment by alexfrom0815** [comment]
Hello, it seems that the current clues are difficult to intuitively find out where the problem is. But I think maybe you can try to increase the number of leaf nodes used when training or testing, maybe it will help. Best wishes!
